docs/blocks/creating-dynamic-blocks.md

```diff
@@ -113,7 +113,9 @@ There are a few things to notice:
 
 ## Live rendering in Gutenberg editor
 
-Gutenberg 2.8 added the [`<ServerSideRender>`](https://github.com/WordPress/gutenberg/tree/master/packages/components/src/server-side-render) block which enables all the rendering to take place on the server using PHP rather than in JavaScript. Server-side render is meant as a fallback; client-side rendering in JavaScript is the preferred implementation.
+Gutenberg 2.8 added the [`<ServerSideRender>`](https://github.com/WordPress/gutenberg/tree/master/packages/components/src/server-side-render) block which enables all the rendering to take place on the server using PHP rather than in JavaScript. 
+
+> Server-side render is meant as a fallback; client-side rendering in JavaScript is the preferred implementation.
```

Comment:
This is the markdown for a block quote, which I don't think this content fits into. I've probably said something like that before, but this is probably better as just a bold/italic paragraph.